.github/workflows/buildAndTest.yml

```diff
@@ -1,10 +1,10 @@
-name: Nightly Build
+name: Build and Test
```

Comment:
Renamed since we use this on nightly build & commit workflow & publish workflow.